strawberryfields/gbs/similarity.py

```diff
@@ -201,3 +210,161 @@ def event_to_sample(photon_number: int, max_count_per_mode: int, modes: int) ->
             available_modes.remove(j)
 
     return sample
+
+
+def orbit_cardinality(orbit: list, modes: int) -> int:
+    """Gives the number of samples belonging to the input orbit.

```

Comment:
This confused me momentarily, until I worked out what it meant. Would be worth adding a short hand-written example, which is then verified via the code example below. Would also be nice to say 'number of unique permutations', maybe?